test/src/test/resources/TestTool/IbisDoc/all.html

```diff
@@ -17,7 +17,9 @@
 <a href='JmsListener.html' target='contentFrame'>JmsListener</a>&nbsp;[<a href='https://javadoc.ibissource.org/latest/nl/nn/adapterframework/jms/JmsListener.html' target='contentFrame'>javadoc</a>]<br/>
 <a href='MessageStoreListener.html' target='contentFrame'>MessageStoreListener</a>&nbsp;[<a href='https://javadoc.ibissource.org/latest/nl/nn/adapterframework/jdbc/MessageStoreListener.html' target='contentFrame'>javadoc</a>]<br/>
 <a href='MqttListener.html' target='contentFrame'>MqttListener</a>&nbsp;[<a href='https://javadoc.ibissource.org/latest/nl/nn/adapterframework/extensions/mqtt/MqttListener.html' target='contentFrame'>javadoc</a>]<br/>
+<a href='PullingIfsaProviderListener.html' target='contentFrame'>PullingIfsaProviderListener</a>&nbsp;[<a href='https://javadoc.ibissource.org/latest/nl/nn/adapterframework/extensions/ifsa/jms/PullingIfsaProviderListener.html' target='contentFrame'>javadoc</a>]<br/>
 <a href='PullingJmsListener.html' target='contentFrame'>PullingJmsListener</a>&nbsp;[<a href='https://javadoc.ibissource.org/latest/nl/nn/adapterframework/jms/PullingJmsListener.html' target='contentFrame'>javadoc</a>]<br/>
+<a href='PushingIfsaProviderListener.html' target='contentFrame'>PushingIfsaProviderListener</a>&nbsp;[<a href='https://javadoc.ibissource.org/latest/nl/nn/adapterframework/extensions/ifsa/jms/PushingIfsaProviderListener.html' target='contentFrame'>javadoc</a>]<br/>
```

Comment:
Er zitten nu ook geen SAP/Coolgen/IBM/Tibco modules in de standaard IbisDoc. IFSA hoeft er ook niet in.

Reply:
Als alle IFSA uit syntax 2 mag, dan weet ik een simpele fix. We kunnen hem excluden bij het zoeken van de Spring Beans. Dan  is er geen reden meer om de Ifsa dependency binnen te halen zoals je hierboven becritiseert. We zijn dan ook van een warning af dat IbisProviderListener niet geladen kan worden. Wil je dat?